src/views/poi_panel.dot

```diff
@@ -80,6 +80,48 @@
 
       {{= this.PoiBlocContainer.render(this.poi) }}
 
+      {{
+        let pictures;
+        const id = this.poi.id.replace('pj:', '');
+        const imagesBlock = this.poi.blocks.find((block) => block.type === 'images');
+        if (imagesBlock) {
+          pictures = imagesBlock.images.filter(img => img.url !== this.poi.topImageUrl);
+        }
+      }}
+
+      {{? pictures && pictures.length }}
+        <div class="poi_panel__pictures">
+            <h5 class="poi_panel__pictures_title">
+              <i class="icon-icon_camera"></i>
+              {{= _('Photos')}}
+            </h5>
+            <a href="https://www.pagesjaunes.fr/pros/{{= id}}" target="_blank" rel="noopener noreferrer" style="text-decoration: none">
```

Comment:
The same here, this URL format should not be hardcoded.  
The API provides a `source_url` for each image, that could be used as a link instead.

Reply:
Ahh, i thought we had to link the whole picture block to the poi's page jaune profil